dashboard/test/ui/features/star_labs/share_buttons.feature

```diff
@@ -17,3 +17,19 @@ Feature: Share Buttons
     When I navigate to the shared version of my project
     And I wait until element "#gameButtons" is visible
     And element "#open-workspace" does not exist
+
+  # Making DPad button show up in Game Lab and not in Sprite Lab
+
+  Scenario: Dpad does not appear for Sprite Lab Share
+    Given I am on "http://studio.code.org/projects/spritelab/"
+    And I wait for the page to fully load
+    When I navigate to the shared version of my project
+    And I wait until element "#gameButtons" is visible
+    And element "#studio-dpad-rim" is not displayed
+  
+  Scenario: Dpad appears for Sprite Lab Share
```

Comment:
nit: Game Lab

Reply:
Thanks for catching this! updated here: https://github.com/code-dot-org/code-dot-org/commit/bb53155ce9b52ee6445b5c8b13ce40f57e766417